mantidproject/mantid
# Elemental analysis line colour

**Description of work.**

When plotting peak lines of different elements, each element will use a different colour.
When all the colours are used, it will loop back and start from the first one.

**To test:**
 - Open elemental analysis
 - Load a data file (2695 should work)
 - Plot peaks for multiple elements

Fixes #25218 
---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
